pkgs/tools/networking/atinout/default.nix

```diff
@@ -4,8 +4,8 @@ stdenv.mkDerivation rec {
   name = "atinout-${version}";
   version = "0.9.2-alpha";
 
-  NIX_CFLAGS_COMPILE = "-Werror=implicit-fallthrough=0";
-  LANG = "C.UTF-8";
+  NIX_CFLAGS_COMPILE = lib.optionalString (!stdenv.cc.isClang) "-Werror=implicit-fallthrough=0";
+  LANG = if stdenv.isDarwin then "en_US.UTF-8" else "C.UTF-8";
```

Comment:
```suggestion
  LC_LANG = if stdenv.isDarwin then "en_US.UTF-8" else "C.UTF-8";
```
Maybe that works

Reply:
LC_ALL works, instead of LANG, if that's better? But it still breaks without the if-statement.

Reply:
Then I have no clue how to make it work without the if.